bedrock/mozorg/templates/mozorg/partnerships.html

```diff
@@ -40,14 +40,11 @@ <h1 class="large">{{ _('Partner with Mozilla') }}</h1>
   <img src="{{ static('img/mozorg/partnerships/icon-firefox.png') }}" alt="{{ _('Firefox icon') }}" class="icon" />
   <h3>{{ _('Firefox for Desktop') }}</h3>
   <p>
-    {{ _('Opportunities include promoted content, distribution, customized versions and more of our PC, Mac &amp; Linux browser.') }}
+    {{ _('Opportunities include distribution, customized versions and more of our PC, Mac &amp; Linux browser.') }}
   </p>
 
   <ul>
     <li>
-      <a href="{{ url('firefox.tiles') }}">{{ _('Learn more about promoted content in Mozilla Firefox') }}</a>
-    </li>
-    <li>
       <a href="{{ url('mozorg.partnerships-distribution') }}">{{ _('Learn more about Mozilla Firefox distribution policies') }}</a>
     </li>
   </ul>
```

Comment:
Can remove all form inputs with `tiles-field` class, as they will now never be displayed.